site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+
+Let's dig in.
+
+## Analyze
+
+One place to start is to use a lab tool like WebPageTest like shown above, so you can compare waterfalls—and LCP—with and without SXG prefetch.
+
+{% Aside 'caution' %}
+[Lab results may differ](https://web.dev/lab-and-field-data-differences/) from real user experiences, but they are a useful tool to diagnose what needs optimization and iterate quickly. We'll cover how to measure the resultant field data in the [Measure](#measure) section.
+{% endAside %}
+
+Generate a test without SXG in the standard way: go to [WebPageTest](https://www.webpagetest.org/) and enter a URL.
+
+  - Consider doing this while signed in, so you can more easily generate a comparison.
+  - Go to "Advanced Configuration" and choose a set of options there, rather than "Simple Configuration". (This makes it easy to assure that you have the same set of parameters as the "with SXG" test, which will use Advanced.)
+  - Under Browser, choose an Android phone under "Chrome Device Emulation". (This is also for parity with the "with SXG" test. As of writing, Google Search only supports SXG on Android.)
+  - Everything else is up to you. It may be helpful to increase "Number of Tests to Run" to 7, and pick a 4G connection.
+
+Generate a test with SXG by starting with the same steps as above, but then go to the Script tab and enter a [WebPageTest script](https://docs.webpagetest.org/scripting/) like the following. This script overrides the URL entered in the main input field:
+
+```clike
+// Disable log collection for the first step. We only want the waterfall for the target navigation.
+logData 0
+
+// Visit a search result page that includes your page.
+navigate https://google.com/search?q=site%3Asigned-exchange-testing.dev+image
+
+// Wait for the prefetch to succeed.
+sleep 10
+
+// Re-enable log collection.
+logData 1
+
+// Navigate to the prefetched SXG on the Google SXG Cache.
+navigate https://signed--exchange--testing-dev.webpkgcache.com/doc/-/s/signed-exchange-testing.dev/sxgs/valid-image-subresource.html
+```
+
+If your page doesn't appear in any Google Search results yet, you can use [this prefetch page](https://signed-exchange-testing.dev/prefetch/) to generate a pretend search results page for this purpose.
+
+
+{% Aside %}
+Note that when Google Search instructs a user's browser to prefetch your SXG, it does so via a URL on webpkgcache.com in order to [avoid revealing the user's intent](https://wicg.github.io/webpackage/draft-yasskin-wpack-use-cases.html#name-privacy-preserving-prefetch) before they navigate. These URLs don't affect the display or behavior of the page, because the browser trusts the original URL that has been signed into the SXG.
```

Comment:
```suggestion
When Google Search instructs a user's browser to prefetch your SXG, it does so via a URL on webpkgcache.com in order to [avoid revealing the user's intent](https://wicg.github.io/webpackage/draft-yasskin-wpack-use-cases.html#name-privacy-preserving-prefetch) before they navigate. These URLs don't affect the display or behavior of the page, because the browser trusts the original URL that has been signed into the SXG.
```